src/kbmod/search/common.h

```diff
@@ -1,9 +1,14 @@
 #ifndef COMMON_H_
 #define COMMON_H_
 
+#include <assert.h>
 #include <string>
+
 #include "pydocs/common_docs.h"
 
+// assert(condition, message if !condition)
+#define assertm(exp, msg) assert(((void)msg, exp))
+
 namespace search {
 #ifdef HAVE_CUDA
 constexpr bool HAVE_GPU = true;
```

Comment:
As discussed in the meeting, let's drop `PixelPos` (this can be in a later PR though)

Reply:
I think it will have to go later. `PixelPos` can be replaced with `Point` directly and deleted, but it would require reorganizing the `geom.h` because `nvcc` complains about certain features used in it so it can't directly be included into the `common.h` at the moment. Splitting that up into `indexing` (indexing + functions) and `geom` (some functions point + trajectory) makes it so that nvcc doesn't have to include the code it can't compile. 

So I'm in favor of making this change, but I got to figure out what is nvcc complaining about.

Reply:
Sounds good to hold off on dropping `PixelPos` for now.